docker/index.yaml

```diff
@@ -65,7 +65,7 @@ artifactory.algol60.net/csm-docker/stable:
     - 0.2.0
 
     iuf:
-    - v0.1.11
+    - v0.1.12
```

Comment:
Are you intending to bring CASMINST-5612 in with this change as well?   If so, we need to add that to the CASMREL for beta.27.

Reply:
Looks like this fails the build because the docker 0.1.12 docker image is not being built.   
This reference in the iuf-container repo still has 0.1.11.
https://github.com/Cray-HPE/iuf-containers/blob/main/.github/workflows/iuf-container.yaml#L36

Reply:
v0.1.12 is already part of release/1.5 in docs-csm -- see here https://github.com/Cray-HPE/docs-csm/blame/release/1.5/workflows/templates/base/iufBase.template.argo.yaml#L68 

There is a PR by @dzou-hpe that does the same thing -- https://github.com/Cray-HPE/csm/pull/2596

Since my changes include switching to v0.1.12 as part of my CASMTRIAGE-5738 ticket, I was being explicit here.

I am not sure why iuf-container is not built as 0.1.12. Let me check that from the CASMINST-5612 PRs...

Reply:
ok so it looks like the iuf-container image didn't get the version update because the github action was not updated. Putting this PR in draft mode and fixing that first.

Reply:
waiting on https://github.com/Cray-HPE/iuf-containers/pull/24

Reply:
Di's PR hasn't been merged yet and CASMINST-5612 did not have CASM-Release-Ready label so it was not pulled in during the release meeting today.   That's the reason why I wanted to be sure it was intended.   I added the label so CASMINST-6512 gets discussed and linked retroactively in tomorrow's meeting.